lib/components/map/default-map.js

```diff
@@ -145,7 +145,7 @@ class DefaultMap extends Component {
           const overlayCompany = oConfig.companies[0] // TODO: handle multi-company overlays
           if (added.includes(overlayMode)) {
             // Company-based mode was just selected; enable overlay iff overlay's company is active
-            if (newQuery.companies.includes(overlayCompany)) {
+            if (newQuery.companies && newQuery.companies.includes(overlayCompany)) {
```

Comment:
You can also use the shorthand notation below:
```suggestion
            if (newQuery.companies?.includes(overlayCompany)) {
```